locales/en/messages.json

```diff
@@ -3891,6 +3891,13 @@
     "pidTuningGyroLowpass": {
         "message": "Gyro Lowpass 1"
     },
+    "pidTuningGyroLowpassHelp": {
+        "message": "The first of two Gyro lowpass filters.<br /><br />The use of gyro lowpass 1 in dynamic mode was important with the earlier dynamic notch code.  Gyro lowpass 1 can usually be disabled completely in firmware 4.3 when RPM filtering is used, unless the gyro has no internal hardware filtering.<br /><br />In dynamic mode, filtering will be stronger at low throttle, with less filtering and less delay as throttle increases. The transition from low to high cutoff will happen earlier, as throttle increases, with higher Gyro lowpass expo values.<br /><br />In static mode, the cutoff frequency is fixed.<br /><br />Filter type defaults to PT1. Higher order filtering is rarely required.",
+        "description": "Help icon for the Gyro Lowpass 1 Filter"
+    },
+    "pidTuningGyroLowpassFrequency": {
+        "message": "Gyro Lowpass 1 frequency [Hz]"
```

Comment:
It is wrong since we are not using the long name in these cases, please remove "Gyro Lowpass 1"
Maybe we should name this "Static frequency [Hz] ??

Reply:
Thank you!

So... I removed the above message definitions, and in html, applied `pidTuningCutoffFrequency` in place of  `pidTuningGyroLowpassFrequency `.  It now says, "Static cutoff [Hz]", because it is static.  I named all such static cutoffs the same way.

The original problem was that someone removed the name `pidTuningGyroLowpassFrequency ` from locales, but didn't remove it from the html, so it was throwing an error.

Reply:
Ouch, maybe i have removed it. Sorry